src/lib/evil/evil_time.h

```diff
@@ -3,6 +3,7 @@
 
 #include <evil_api.h>
 
+#include <check.h>
```

Comment:
Wait what

Reply:
This shouldn't be here. At all. Check is for test-units, why is it here?

Reply:
hahaha he is just testing the issue of having both gettimeofday

Reply:
oh no wrong branch

Reply:
>:(

Reply:
Damn, markdown turned my angry-face into a sad face